Doc/library/zipimport.rst

```diff
@@ -44,9 +44,6 @@ doesn't contain :file:`.pyc` files, importing may be rather slow.
       follows the specification in :pep:`273`, but uses an implementation written by Just
       van Rossum that uses the import hooks described in :pep:`302`.
```

Comment:
If the PEP 302 "see also" is being dropped (which seems like a good idea), does it make sense to keep the even older reference to PEP 273? Perhaps both of them should be replaced with cross references to importlib and the import section in the language reference?

Reply:
I kept the zip one since it's specific to this module. I'm leaning on the fact that the importlib-related methods reference importlib itself. But a direct reference to importlib probably wouldn't hurt. I don't think a language reference link is important, though, since this module is more about implementing importlib APIs and people can go there if they want language-level details.